experiment/assignment/quiz.php

```diff
@@ -6,12 +6,12 @@
 
 $starttime = (!isset($_POST['starttime']) ? time() : $_POST['starttime']);
 $answers = (!isset($_POST['answers']) ? "0 0 0" : $_POST['answers']);
-$answer = (isset($_POST['quiz']) ? $_POST['quiz']: 0);
+$answer = ($_POST['quiz'] ?? 0);
```

Comment:
Some discrepancy in the generated code. One line is converted to the new syntax, the other is not.